tests/_support/AcceptanceTester.php

```diff
@@ -33,4 +33,129 @@ public function waitForPageTitle($title, $timeout = 60)
 		$I = $this;
 		$I->waitForText($title, $timeout, ['class' => 'page-title']);
 	}
+	
+	/**
+	 * Function to check for PHP Notices or Warnings
+	 *
+	 * @param string $page Optional, if not given checks will be done in the current page
+	 *
+	 * @note: doAdminLogin() before
+	 */
+	public function checkForPhpNoticesOrWarnings($page = null)
+	{
+		$I = $this;
+
+		if($page)
+		{
+			$I->amOnPage($page);
+		}
+
+		$I->dontSeeInPageSource('Notice:');
+		$I->dontSeeInPageSource('<b>Notice</b>:');
+		$I->dontSeeInPageSource('Warning:');
+		$I->dontSeeInPageSource('<b>Warning</b>:');
+		$I->dontSeeInPageSource('Strict standards:');
+		$I->dontSeeInPageSource('<b>Strict standards</b>:');
+		$I->dontSeeInPageSource('The requested page can\'t be found');
+	}
+
+	/**
+	 * Function to select Toolbar buttons in Joomla! Admin Toolbar Panel
+	 *
+	 * @param   string  $button  The full name of the button
+	 */
+	public function clickToolbarButton($button)
+	{
+		$I = $this;
+		$input = strtolower($button);
+
+		// @todo Needs to find way to work with different window size.
+		/*$screenSize = explode("x", $this->config['window_size']);
```

Comment:
@pritalpatel You have to move / copy the window_size in the configuration (acceptance.suite.yml) from JoomlaBrowser to AcceptanceHelper.

See https://github.com/joomla-projects/gsoc16_browser-automated-tests/blob/staging/tests/acceptance.suite.dist.yml#L16